### PR TITLE
Update NCSA access log optional 'enabled' attribute documentation

### PR DESCRIPTION
**Pull request by mrsaldana**

### CODE REVIEW

**Comment by mrsaldana** [comment]
fixes #11123

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -156,7 +156,7 @@ log.access=HTTP Access Logging
 log.access.desc=HTTP access logs contain a record of all inbound HTTP client requests.
 
 log.access.enabled=Enabled
-log.access.enabled.desc=Enable access logging.
+log.access.enabled.desc=Enable access logging when accessLogging configuration element is defined. Note: The access logging needs to be configured for this attribute to take effect.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: change
Enable
to
Enables

change
when accessLogging configuration
to
when the accessLogging configuration

change
needs to be configured
to
must be configured

change
Note: The access logging
to
Note: Access logging

Comment (mrsaldana) [reply dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: Updated!

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#run-libby-bot

**Pull request merged**